book/src/faqV2.md

```diff
@@ -0,0 +1,487 @@
+
+In short, it is nothing to worry about.
+
+### I see beacon node logs `Head is optimistic`, and I am missing attestations. What should I do?
+
+The log looks like:
+
+```
+WARN Head is optimistic       execution_block_hash: 0x47e7555f1d4215d1ad409b1ac188b008fcb286ed8f38d3a5e8078a0af6cbd6e1, info: chain not fully verified, block and attestation production disabled until execution engine syncs, service: slot_notifier
+```
+
+It means the beacon node will follow the chain, but it will not be able to attest or produce blocks. This is because the execution client is not synced, so the beacon chain cannot verify the authenticity of the chain head, hence the word `optimistic`. What you need to do is to make sure that the execution client is up and syncing. Once the execution client is synced, the error will disappear.
+
+### My beacon node logs `CRIT Beacon block processing error error: ValidatorPubkeyCacheLockTimeout, service: beacon`, what should I do? 
+
+An example of the log is shown below:
+
+```
+CRIT Beacon block processing error           error: ValidatorPubkeyCacheLockTimeout, service: beacon
+WARN BlockProcessingFailure                  outcome: ValidatorPubkeyCacheLockTimeout, msg: unexpected condition in processing block.
+```
+
+A `Timeout` error suggests that the computer may be overloading at the moment, for example, the execution client is still syncing. You may use the flag `--disable-lock-timeouts` to silence this error, although it will not fix the underlying slowness. Nevertheless, this is a relatively harmless log, and the error should go away once the resources used are back to normal. 
+
+### My beacon node logs `WARN BlockProcessingFailure outcome: MissingBeaconBlock`, what should I do?
+
+An example of the full log is shown below:
+
+```
+WARN BlockProcessingFailure                  outcome: MissingBeaconBlock(0xbdba211f8d72029554e405d8e4906690dca807d1d7b1bc8c9b88d7970f1648bc), msg: unexpected condition in processing block.
+```
+
+`MissingBeaconBlock` suggests that the database has corrupted. You should wide the database and use [Checkpoint Sync](./checkpoint-sync.md) to resync the beacon chain. 
+
+### After checkpoint sync, the progress of `downloading historical blocks` is slow. Why?
+
+This is a normal behaviour. Since [v4.1.0](https://github.com/sigp/lighthouse/releases/tag/v4.1.0), Lighthouse implements rate-limited backfill sync to mitigate validator performance issues after a checkpoint sync. This is not something to worry because backfill sync / historical data is not required for staking. However, if you opt to sync the chain as fast as possible, you can add the flag `--disable-backfill-rate-limiting` to the beacon node.
```

Comment:
```suggestion
This is a normal behaviour. Since [v4.1.0](https://github.com/sigp/lighthouse/releases/tag/v4.1.0), Lighthouse implements rate-limited backfill sync to mitigate validator performance issues after a checkpoint sync. This is not something to worry about since backfill sync / historical data is not required for staking. However, if you opt to sync the chain as fast as possible, you can add the flag `--disable-backfill-rate-limiting` to the beacon node.
```